### PR TITLE
[FIX] website, *: fix website content list / kanban views

### PR DESCRIPTION
**Pull request by xO-Tx**
*: website_blog, website_forum, website_hr_recruitment, website_sale, website_slides

The goal of this PR is to add some fixes on website "pages" list / kanban views after [1].

[1]: https://github.com/odoo/odoo/pull/101319

Related to task-2889981

### CODE REVIEW

**Comment by xO-Tx** [comment]
@qsm-odoo I added the fixes related to kanban views here, you can merge & fw if OK for you :+1:, just a remark about the website dropdown not working for 'Appointments':
Actually it's not working because `appointment.type` is not linked to a website (doesn't inherit `website.multi.mixin`) so it shouldn't have a website filter on searchview... I disabled that on the [first PR](https://github.com/odoo/odoo/pull/98941/commits/5bbb78dd2f653fd21385803feac6e2d73daf25c3)

File: addons/website_hr_recruitment/views/website_pages_views.xml [hunk 1]
```diff
@@ -12,13 +12,12 @@
...
             <attribute name="action">open_website_url</attribute>
-
-            <!-- TODO could be a nice addition but editing "Department" crashes... -->
-            <!-- <attribute name="multi_edit">1</attribute> -->
+            <attribute name="multi_edit">1</attribute>
         </xpath>
 
         <field name="name" position="after">
             <field name="website_url"/>
+            <field name="company_id" invisible="1"/>
```
Comment (qsm-odoo) [inline addons/website_hr_recruitment/views/website_pages_views.xml:20]: Was not there already a company_id in the base view?

**Review by qsm-odoo — CHANGES_REQUESTED** [review]
> Actually it's not working because appointment.type is not linked to a website (doesn't inherit website.multi.mixin) so it shouldn't have a website filter on searchview... I disabled that on the [first PR](https://github.com/odoo/odoo/pull/98941/commits/5bbb78dd2f653fd21385803feac6e2d73daf25c3)

Mmh strange, I guess I tested wrong. Good then :+1:

File: addons/website_hr_recruitment/views/website_pages_views.xml (hunk 1 above)
Comment (xO-Tx) [reply addons/website_hr_recruitment/views/website_pages_views.xml:20]: @qsm-odoo sorry, I forgot to explain that on commit message :man_facepalming:
 
If user doesn't match the `groups` attribute, the field **with this attribute** will be removed from the view (not loaded/available for `domain`, `attrs`... computing) so if `company_id` is not added as "invisible", the edit of `department_id` (using it on its domain) will crash outside of multi company.

You can find a good explanation in [DLE's view refactoring](https://github.com/odoo/odoo/pull/95729#issuecomment-1257954879).

Comment (qsm-odoo) [reply addons/website_hr_recruitment/views/website_pages_views.xml:20]: Ok, good to know, I was not sure duplicating was the way to go :+1:

**Review by qsm-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Pull request closed without merging**